models/demos/t3000/falcon40b/tt/falcon_mlp.py

```diff
@@ -89,27 +89,29 @@ def __init__(
                 self.dense_4h_to_h_weights.append(
                     dense_4h_to_h_weights_host.to(devices[i], self.model_config["DENSE_4H_TO_H_MM_WEIGHTS_MEMCFG"])
                 )
-                tt_lib.tensor.dump_tensor(
+                ttnn.experimental.tensor.dump_tensor(
                     str(dense_4h_to_h_path),
                     dense_4h_to_h_weights_host,
                 )
 
     def set_model_config(self, model_config):
         self.model_config = model_config
 
-    def __call__(self, x: List[tt_lib.tensor.Tensor], llm_mode: str) -> List[tt_lib.tensor.Tensor]:
+    def __call__(
+        self, x: List[ttnn.experimental.tensor.Tensor], llm_mode: str
+    ) -> List[ttnn.experimental.tensor.Tensor]:
         if llm_mode == "prefill":
             return self.fwd_prefill(x)
         elif llm_mode == "decode":
             return self.fwd_decode(x)
         else:
             assert False
 
-    def fwd_decode(self, x: List[tt_lib.tensor.Tensor]) -> List[tt_lib.tensor.Tensor]:
+    def fwd_decode(self, x: List[ttnn.experimental.tensor.Tensor]) -> List[ttnn.experimental.tensor.Tensor]:
```

Comment:
Can we use `ttnn.Tensor` instead of `ttnn.experimental.tensor.Tensor`?

Reply:
I agree (ttnn.tensor is the same as ttnn.experimental), just wanted to move everything to ttnn.experimental first and later change to ttnn in another PR since this is only renaming issue and moving to ttnn might have some differences. For the sake of consistency wanted to keep them separate.

Reply:
That makes a lot of sense for all ops, since changing to ttnn ops might cause issues/differences, but I think ttnn.Tensor is also just an alias for the same tensor structure, right? So underneath that's all the same and it's really just about renaming.